tests/image/embedding/test_model.py

```diff
@@ -51,7 +51,7 @@ def test_load_from_checkpoint_dependency_error():
 
 
 @pytest.mark.skipif(not (_TORCHVISION_AVAILABLE and _VISSL_AVAILABLE), reason="vissl not installed.")
-@pytest.mark.parametrize("backbone, training_strategy", [("resnet", "barlow_twins")])
+@pytest.mark.parametrize("backbone, training_strategy", [("vision_transformer", "dino")])
```

Comment:
This and other changes in the tests will be reverted, and we'll instead pass multiple args to test for multiple backbones and pretraining transforms. For now, relying on CI to give results as local GPU isn't working.